lint/action.yaml

```diff
@@ -20,10 +20,15 @@ runs:
       if: hashFiles('package-lock.json') != ''
       shell: bash
       run: npm ci
-    - name: Install actionlint
-      uses: KeisukeYamashita/setup-release@v1.0.2
-      with:
-        repository: rhysd/actionlint
+    - name: Install actionlint via bash
+      shell: bash
+      run: |
```

Comment:
LGTM, I'd only add a reference to where we can find these install instructions for future maintainers 😄